src/containers/categories/categories.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {Component} from 'react';
+import {Text, View, StyleSheet} from 'react-native';
+import {Header, BackButton, NorraButton, BoxImage} from '@components';
+import {PRIMARY_NAVY_BLUE, CategoryPageMsg} from '@modules';
+
+interface ICategState {
+  options: Array<string>;
+  selectedOpt: Array<string>;
+}
+
+export class Categories extends Component<any, ICategState> {
+  state = {
+    options: ['Sports', 'Hygiene', 'Healthy Diet'],
+    selectedOpt: [''],
+  };
+
+  optionClickHandler = (item: string) => {
+    const {selectedOpt} = this.state;
```

Comment:
Can we shorten this function by using the below logic?

`if(item exists)`
   `remove from list`
`else`
    `add item`
`setState()`